### PR TITLE
Prevent slack send from failing release build

### PR DESCRIPTION
**Pull request by Mpdreamz**

### CODE REVIEW

**Review by stevejgordon — DISMISSED** [review]
LGTM

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -97,14 +97,14 @@ jobs:
...
 
     - name: Attach Profiler And Startup Hooks
-      continue-on-error: true #continue for now until we see it working in action
       env:
         GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
         gh release upload ${{ github.ref_name }} "${{ env.PREFIX_APM_AGENT }}${{ steps.bootstrap.outputs.agent-version }}${{ env.SUFFIX_APM_AGENT }}" "${{ env.PREFIX_APM_PROFILER }}${{ steps.bootstrap.outputs.agent-version }}${{ env.SUFFIX_APM_PROFILER }}"
 
     - if: ${{ success() }}
       uses: elastic/oblt-actions/slack/send@v1.7.0
```
Comment (v1v) [inline .github/workflows/release.yml:106]:
```suggestion
      uses: elastic/oblt-actions/slack/send@v1
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -115,6 +115,7 @@ jobs:
 
     - if: ${{ failure() }}
       uses: elastic/oblt-actions/slack/send@v1.7.0
```
Comment (v1v) [inline .github/workflows/release.yml:117]:
```suggestion
      uses: elastic/oblt-actions/slack/send@v1
```

**Review by v1v — COMMENTED** [review]
Latest version does some message manipulation to avoid those issues :/

Sorry!

**Review by v1v — APPROVED** [review]

**Pull request merged**